include/accessor.h

```diff
@@ -154,7 +154,7 @@ class accessor<DataT, Dims, Mode, target::device> : public detail::accessor_base
 	accessor(accessor&&) noexcept = default;
 	accessor& operator=(accessor&&) noexcept = default;
 
-#if !defined(__SYCL_DEVICE_ONLY__)
+#if !CELERITY_DETAIL_COMPILE_TIME_TARGET_DEVICE_ONLY // host- or universal single pass
```

Comment:
This only works because ACPP does not actually check `is_device_copyable` (which implies `trivially_copyable`) - consider adding a comment!